app/services/commonApiService.ts

```diff
@@ -106,6 +99,73 @@ export const onSponsorApiSuccess = async (response: any, dispatch: any, navigati
   console.log("allDatatoStore ",prevPage,"--",allDatatoStore);
   navigation.navigate('Terms');
 }
+export const onCountryApiSuccess = async (response: any, dispatch: any, navigation: any,languageCode: string,prevPage:string):Promise<any> => {
+  console.log('Response for country is', response)
+  if (response && response[0] && response[0].apiEndpoint == appConfig.countryGroups) {
+    response = response[0];
+   
+    if(response.data && response.data.status && response.data.status == 200)
+    {
+      console.log('type of sponser data for country is', typeof response.data.data)
+      console.log('sponser data for country is',  response.data.data)
+      console.log('sponser data for country issdsd',  response.data?.data[0])
+      dispatch(setuserIsFirstTime(true));
+      const allDatatoStore = await getAllDataToStore(languageCode,dispatch,prevPage);
+      console.log("allDatatoStore ",prevPage,"--",allDatatoStore);
+      //navigation.push('Localization'); 
```

Comment:
Should we keep this code commented? If the code is finalized, you can remove the commented sections